src/contentful/contentful.service.ts

```diff
@@ -25,20 +25,21 @@ const CALLS = 8
 
 @Injectable()
 export class ContentfulService {
-  async getAllEntries(): Promise<Entry[]> {
-    const allEntryCollections: EntryCollection<EntrySkeletonType>[] = []
-
+  async getAllEntries() {
+    const allEntryCollections: EntryCollection<EntrySkeletonType>[] = [];
     for (let i = 0; i < CALLS; i++) {
       const entryCollection = await contentfulClient.getEntries({
         limit: LIMIT,
+        include: 5,
         skip: i * LIMIT,
-      })
-      allEntryCollections.push(entryCollection)
+      });
```

Comment:
You need to get `eslint` running automatically in your environment. You're sprinkling tabs, spaces, and `;`s all over the place that'll eventually break the build once we get around to enforcing it.

<img width="841" alt="image" src="https://github.com/user-attachments/assets/64307d01-05d5-43ec-a7ac-0efed3a0f81c" />